cmd/minikube/cmd/start.go

```diff
@@ -432,13 +432,23 @@ func displayEnviron(env []string) {
 	}
 }
 
-func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion string, machineName string) error {
+func showKubectlInfo(kcs *kubeconfig.Settings, k8sVersion, machineName string) error {
 	if k8sVersion == constants.NoKubernetesVersion {
 		register.Reg.SetStep(register.Done)
 		out.Step(style.Ready, "Done! minikube is ready without Kubernetes!")
-		out.BoxedWithConfig(box.Config{Py: 1, Px: 4, Type: "Round", Color: "Green"}, style.Tip, "Things to try without Kubernetes ...", `- "minikube ssh" to SSH into minikube's node.
+
+		// Runtime message.
+		msg := `- "minikube ssh" to SSH into minikube's node.`
+		switch viper.GetString(containerRuntime) {
+		case constants.DefaultContainerRuntime:
+			msg += `
 - "minikube docker-env" to point your docker-cli to the docker inside minikube.
-- "minikube image" to build images without docker.`)
+- "minikube image" to build images without docker.`
+		case constants.CRIO:
+			msg += `
+- "minikube podman-env" to point your podman-cli to the podman inside minikube`
+		}
+		out.BoxedWithConfig(box.Config{Py: 1, Px: 4, Type: "Round", Color: "Green"}, style.Tip, "Things to try without Kubernetes ...", msg)
```

Comment:
I suggest we Dupliclate code here for sake of translations 
```
// because the translation extract (make extract) needs simple strings to be included in translations we have to pass simple strings)
```
anything that is printed with out package gets added to our translation.json file for translators to translate.

so it is better to have three out.BoxwithConfig with full english text than trying to do the "right code" thing